coral-component-table/src/scripts/Table.js

```diff
@@ -2467,6 +2467,10 @@ const Table = Decorator(class extends BaseComponent(HTMLTableElement) {
       headerCell.setAttribute('role', 'rowheader');
     }
     headerCell.setAttribute('scope', scope);
+      
+    if(headerCell.hasAttribute('sortable')){
+      headerCell._elements.content.setAttribute('role', 'button');
```

Comment:
I think `headerCell.content` also points to `headerCell._elements.content`. Please change to `headerCell.content`

Reply:
Hi @Pareesh I made the change